src/indigo/less/indigo.less

```diff
@@ -97,6 +97,10 @@ html {
 
 body {
     padding-top: 60px;
```

Comment:
jak tedy s 'min-width: 480px', pokud  to teda nebudeme implementovat, tak aspon napis nejaky duvod proc si myslis ze to tam nepatri .] Ja v tom zadny blok nevidim. Naopak budeme na bezpecnejsi strane

Reply:
Dal jsem si chrome na 390px šířku a zkusil udělat pár screenů. První a druhý je současné nastavení. Poslední je s podmínkou "@media (min-width: 420px) and (max-width: 768px)". Pak se to spíše rozbije jak jsem psal. Plus nefunguje ani toggle (to by se asi nějak udělalo).

![mobile_close](https://user-images.githubusercontent.com/12331181/46733922-040c3300-cc92-11e8-81e9-233759813552.png)
![mobile_open](https://user-images.githubusercontent.com/12331181/46733923-040c3300-cc92-11e8-9e29-715feb34af44.png)
![mobile_min_width_420](https://user-images.githubusercontent.com/12331181/46733924-040c3300-cc92-11e8-93f0-67de5e9ad135.png)

Reply:
Myslim ze si porad nerozumime, myslim to obecneji.
Jde mi o to dat viewportu nejaky limit, jinak se uoptimalizujeme. Od urcity min sirky to uz bude scrollovat
Jak jsem psal tady. Styl na body
https://github.com/keboola/kbc-ui/pull/2099#discussion_r223985291
Ok, 480 je mozna zbytecne moc, dejme treba 360px

hele:
![tf1](https://user-images.githubusercontent.com/15363559/46741222-d8df0f00-cca4-11e8-9e64-2bb1a82c7dd8.gif)

Reply:
Jo už asi rozumím :) tak to doplním.